src/System.Runtime/tests/System/Runtime/ExceptionServices/HandleProcessCorruptedStateExceptions.cs

```diff
@@ -38,12 +38,12 @@ static void CauseAVInNative()
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)] // Feature Corrupting Exceptions not present for Linux
         [ActiveIssue("https://github.com/dotnet/corefx/issues/21123", TargetFrameworkMonikers.Uap)]
-        [ActiveIssue(27460)]
         public static void ProcessExit_Called()
         {
             using (RemoteInvokeHandle handle = RemoteInvoke(() => { CauseAVInNative(); return SuccessExitCode; }))
             {
                 Process p = handle.Process;
+                handle.Process = null;
                 p.WaitForExit();
```

Comment:
Are you doing this to avoid validating the exit code? If so it'd be better to use options.CheckExitCode = false when calling RemoteInvoke.

Reply:
The reason I nullified the ```handle.Process``` is because after we de-reference it into variable ```p```, it's lifetime would no longer be controlled by ```handle```.
This would prevent us from sometimes getting [this](https://github.com/dotnet/corefx/blob/d27d0d98b4cffbe8b15c5acdb2403112029c48d0/src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.cs#L212) assertion failure when handle is not properly disposed.

Reply:
Why would we get that assertion?  We're disposing of the RemoteInvokeHandle.

Reply:
I think that would happen if RemoteInvokeHandle gets disposed before variable p gets disposed. then the Process is referenced by ```p``` as ```handle``` is getting disposed

Reply:
I don't understand.  That assert has nothing to do with the Process instance.

Reply:
thanks @stephentoub for offline discussion. I'll update the PR in a bit.